tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1192,4 +1192,111 @@ public function testDoesNotListIndexesImplicitlyCreatedByForeignKeys()
         self::assertArrayHasKey('explicit_fk1_idx', $indexes);
         self::assertArrayHasKey('idx_3d6c147fdc58d6c', $indexes);
     }
+
+    /**
+     * @group 2782
+     * @group 6654
+     */
+    public function testComparatorShouldReturnFalseWhenLegacyJsonArrayColumnHasComment()
+    {
+        if ( ! $this->_sm->getDatabasePlatform()->hasNativeJsonType()) {
+            $this->markTestSkipped('This test is only supported on platforms that have native JSON type.');
+        }
+
+        $table = new Table('json_array_test');
+        $table->addColumn('id', 'integer');
```

Comment:
Can you drop columns that aren't strictly required?

Reply:
Sure...